docs/run-taco.md

```diff
@@ -2,24 +2,30 @@
 title: Run Your Packaged Connector (TACO file)
 ---
 
-Starting in 2019.4 Beta 1, you can load packaged connectors (otherwise known as TACO files). To use a connector in earlier Tableau versions, see  [Run Your "Under Development" Connector]({{ site.baseurl }}/docs/share)
+This section describes how to load packaged connectors (otherwise known as TACO files). A TACO file has a .taco filename extension. 
+
+**Note:** To use a connector in a version of Tableau before 2019.4 Beta 1, see  [Run Your "Under Development" Connector]({{ site.baseurl }}/docs/share)
 
 ## Run a packaged connector in Tableau Desktop
-1. Copy your packaged connector file (with a .taco filename extension) into your My Tableau Repository/Connectors directory. 
+1. Copy your TACO file into your My Tableau Repository/Connectors directory. 
 1. Launch Tableau Desktop.
 
+## Run a packaged connector in Tableau Prep
+1. Copy your TACO file into your My Tableau Prep Repository/Connectors directory. 
+1. Launch Tableau Prep.
+
```

Comment:
What version of Tableau Prep supports tacos? If it's not 2019.4, like for desktop, we should mention it here.